occurrences/tests/test_api.py

```diff
@@ -504,7 +527,12 @@ def test_update_occurrence_of_published_event(
 
 
```

Comment:
I think can also add tests for languages query as well

Reply:
because I'm obsessed with getting high test coverage 😜

Reply:
Absolutely! I didn't want to write them before your first look! :) They would have gone to a trash folder if you would have rejected this. That's why I left this PS as a draft.